src/Utility/requests.ts

```diff
@@ -0,0 +1,169 @@
+import t, { t_r } from './i18n';
+import { CriticalException, rethrow } from './errors';
+import { generateReference } from 'letter-generator';
+import { deepCopyObject } from '../Utility/common';
+import type {
+    IdDataElement,
+    Address,
+    AccessRequest,
+    RequestType,
+    DataField,
+    Request,
+    CustomTemplateName,
+} from '../types/request';
+import type { Company, RequestLanguage, SupervisoryAuthority } from '../types/company';
+
+export const REQUEST_ARTICLES = { access: '15', erasure: '17', rectification: '16', objection: '21(2)' };
+export const REQUEST_FALLBACK_LANGUAGE = 'en'; // We'll use English as hardcoded fallback language
+
+export function isAddress(value: IdDataElement['value']): value is Address {
+    return (value as Address).country !== undefined;
+}
+
+export function isSva(sva: unknown): sva is SupervisoryAuthority {
+    return (sva as SupervisoryAuthority)['complaint-language'] !== undefined;
```

Comment:
No casting necessary.

```suggestion
    return !!sva && typeof sva === 'object' && 'complaint-language' in sva;
```